eng/Versions.props

```diff
@@ -155,6 +155,6 @@
       in Directory.Packages.props - Microsoft.TestPlatform.TranslationLayer and Microsoft.TestPlatform.ObjectModel.
       This version needs to match the Test SDK version consumed by Arcade.
     -->
-    <MicrosoftNETTestSdkVersion>17.5.0</MicrosoftNETTestSdkVersion>
+    <MicrosoftNETTestSdkVersion>17.13.0</MicrosoftNETTestSdkVersion>
```

Comment:
@NikolaMilosavljevic - we need to take this test platform update because otherwise our tests will stop running in CI (see https://github.com/dotnet/sdk/issues/49248)

I don't understand what the comment above means - is this going to break anything in the VMR?

Reply:
> will stop

Correction: have stopped

Reply:
> we need to take this test platform update because otherwise our tests will stop running in CI

I'm surprised that this update is required for running tests. I think updating xunit.runner.console should just be sufficient.

It's still a good idea to update Microsoft.NET.Test.Sdk for sure.

Reply:
No, we cannot. xunit.runner.visualstudio depends on Microsoft.TestPlatform.ObjectModel (>= 17.13.0) on framework.

Reply:
Oh, I didn't realize xunit's dependency.